paradise_papers_search/paradise_papers_search/constants.py

```diff
@@ -0,0 +1,9 @@
+from neomodel import db
+
+countries = db.cypher_query("MATCH (n) WHERE NOT n.countries CONTAINS ';' RETURN DISTINCT 'node' as entity, n.countries AS countries UNION ALL MATCH ()-[r]-() WHERE EXISTS(r.countries) RETURN DISTINCT 'relationship' AS entity, r.countries AS countries")[0]
```

Comment:
this query could be shorter and faster lie : `MATCH (n) WHERE NOT n.countries CONTAINS ';' RETURN DISTINCT  n.countries AS countries` . Could you double check running it through the browser?

Reply:
Also let's check if we can format all cypher queries like:
``` python
countries = db.cypher_query(
    """
    MATCH (n) 
    WHERE NOT n.countries CONTAINS ';'
    RETURN DISTINCT n.countries AS countries
    """
)
```